docs/README.md

```diff
@@ -1,12 +1,14 @@
 # Overview
 
-IoT Identity Service is made up of three services that provide provisioning and cryptographic services:
+Azure IoT Edge brings the analytical power of the cloud closer to devices to drive better business insights and enable offline decision making through the use of [Edge modules](https://docs.microsoft.com/azure/iot-edge/iot-edge-modules).  The full IoT Edge runtime, in concert with IoT Hub, [provides numerous capabilities](https://docs.microsoft.com/azure/iot-edge/iot-edge-runtime) to scale out and manage an IoT solution from the cloud.  
```

Comment:
The rest of the docs don't use two spaces after the period, so please don't use them here.